aten/src/ATen/core/dispatch/Dispatcher.cpp

```diff
@@ -60,10 +75,14 @@ void Dispatcher::deregisterSchema(const OperatorHandle& op) {
     AT_ERROR("Tried to deregister op schema that still has kernels registered");
   }
 
-  // note: call listeners *before* operator is removed, i.e. dispatcher is still valid for removed op
-  listeners_->callOnOperatorDeregistered(op);
+  // reduce refcount and actually deregister if no references left
+  --op.operatorDefIterator_->refcount;
+  if (0 == --op.operatorDefIterator_->refcount) {
```

Comment:
don't you decrease it twice? O_O

in general, would be nice to have a test for it. That being said, as I mentioned earlier, I'm not sure we'd practically use "unloading" part of operators

Reply:
Yes, this already blew up when I tried to use it and it's fixed in the newer version. Unit tests are in the pipeline.